### PR TITLE
Config and other minor changes for running Arctic pipeline in OKE

### PR DESCRIPTION
**Pull request by TomYipOracle**
The config has been tested with the following commands and configs: 

nextflow run TomYipOracle/ncov2019-artic-nf -r oke_testing -with-trace -with-report -with-timeline -with-dag dag.png --readpat '*{1,2}.fastq.gz' --illumina --prefix illumina -profile oke_private --directory /workspace/ncov2019-artic-nf/input --varCaller iVar --refmap '"{}"' --TESToutputMODE true --outdir /workspace/ncov2019-artic-nf/output -latest

nextflow run main.nf -with-trace -with-report -with-timeline -with-dag dag.png --readpat '*{1,2}.fastq.gz' --illumina --prefix illumina -profile oke_private --directory /workspace/ncov2019-artic-nf/input --varCaller iVar --refmap '"{}"' --TESToutputMODE true --outdir /workspace/ncov2019-artic-nf/output

### CODE REVIEW

File: bin/makeReport.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 nextclade['nextcladeVersion']=str(nextclade_version).strip()
 nextclade.set_index('program',inplace=True)
 n=nextclade.to_dict(orient='index')
-with open('nextclade.json','r') as inf:
+with open('nextclade.json','rt', encoding= 'utf-8') as inf:
```
Comment (TomYipOracle) [inline bin/makeReport.py:44]: @JezSw do you recognize this change?

File: modules/analysis.nf [hunk 2]
```diff
@@ -39,9 +39,7 @@ process download_nextclade_files {
     """
     home=PWD
     nextclade_ver=`(nextclade -v)`
-    wget -P nextclade_files https://raw.githubusercontent.com/nextstrain/nextclade/\${nextclade_ver}/data/sars-cov-2/tree.json
-    wget -P nextclade_files https://raw.githubusercontent.com/nextstrain/nextclade/\${nextclade_ver}/data/sars-cov-2/genemap.gff
-    wget -P nextclade_files https://raw.githubusercontent.com/nextstrain/nextclade/\${nextclade_ver}/data/sars-cov-2/qc.json 
+    nextclade dataset get --name 'sars-cov-2' --output-dir nextclade_files
```
Comment (TomYipOracle) [inline modules/analysis.nf:42]: I do not intend to change this. Should I revert back to the previous version?

File: modules/analysis.nf [hunk 3]
```diff
@@ -65,9 +63,7 @@ process nextclade {
     """
     nextclade --input-fasta ${fasta} \
 	--input-root-seq ${reffasta} \
-	--input-gene-map=nextclade_files/genemap.gff \
-	--input-tree=nextclade_files/tree.json \
-	--input-qc-config=nextclade_files/qc.json \
+	--input-dataset=nextclade_files \
```
Comment (TomYipOracle) [inline modules/analysis.nf:66]: I do not intend to change this. Should I revert back to the previous version?

File: bin/makeReport.py (hunk 1 above)
Comment (JezSw) [reply bin/makeReport.py:44]: This was introduced by Nick as part of 347803c6fd9f2f7f5c9f2790b037c370840139bc

File: modules/analysis.nf (hunk 2 above)
Comment (JezSw) [reply modules/analysis.nf:42]: This was introduced by Nick as part of 347803c

File: modules/analysis.nf (hunk 3 above)
Comment (JezSw) [reply modules/analysis.nf:66]: This was introduced by Nick as part of 347803c

**Comment by JezSw** [comment]
I'll merge this for now so we have sight of it. Although it will need further work before being fully integrated into the "normal" SP3 version. @nick297 is working on a revised workflow (https://github.com/oxfordmmm/SARS-CoV2_workflows) so it may be best to fold it into that once we look at deploying that properly, probably in the new year.

**Pull request merged**